crates/bevy_reflect/src/serde/de.rs

```diff
@@ -23,6 +23,7 @@ pub trait DeserializeValue {
 }
 
 trait StructLikeInfo {
+    #[allow(dead_code)]
```

Comment:
@MrGVSV @soqb what do you think should be done about these dead-code warnings?

Reply:
If it's not being used, we can just get rid of it. These traits are private to this one module

Reply:
go ahead and remove them. i don't know whether it was the `TypePath` changes or the serialization format changes which made these methods unnecessary but we don't need the path here anymore.

Reply:
Removed them in the last commit!